### ISSUE TITLE
Table one in r not showing test name

### ISSUE THREAD

**Issue by EddieItelman** [kaz-yos/tableone#63]
Unlike the python package, the r version is not showing the name of the test performed (chi square, ANOVA etc).

nor did I find documentation on  how to make this happen.

**Comment by SimonStolz** [comment]
I agree! "Normal" test is left unspecified and just "blank" by default (?). I am not sure if this is a good choice. _There should be equivalently something written in it (not only for exceptions)._

I would also suggest to _write the specific test's name_ here and provide a clearer overview which R function is behind which test in the _documentation_: Documentation of `CreateTableOne` suggests for normal to be by default "oneway.test", and "is is equivalent of the t-test when there are only two groups.".

However when I attempt:
`set.seed(1)
x <- rnorm(50, 50, 34)
set.seed(2)
y <- rnorm(50, 70, 34)
d <- rbind(cbind(kauf = 1, x),
           cbind(kauf = 0, y)) %>% as.data.frame()`

Then
`CreateTableOne(vars = "x", strata="kauf", data = d)` 
gives me 
> p < 0.006

but  `t.test(d$x, d$y)` gives me
> t = 18.031
> p-value < 0.00000000000000022

Lastly, it would be also great to have _a further column for the test statistic_ (e.g., t-statistic score).
